include/openssl/configuration.h.in

```diff
@@ -76,4 +76,7 @@ extern "C" {
 #ifndef OPENSSL_NO_EC2M
 #define OPENSSL_NO_EC2M
 #endif
+#ifndef OPENSSL_NO_QUIC
```

Comment:
It's there so that applications that use the OPENSSL_NO_QUIC to toggle what QUIC implementation to build agains sense its absence correctly. As for why this file, I think this is the one that produces configuration.h at build time, which logically is where they should live.

Reply:
Oh right.  I forgot about that.  Thanks.